src/types/OAuthConfig.ts

```diff
@@ -1,29 +1,45 @@
 import { OAuthGrantType } from '../types';
 
-type GrantConfigBase = {
+type CredentialsDirConfig = {
   credentialsDir: string;
+};
+
+type PassCredentialsClientConfig = {
```

Comment:
What does the `Passs` meane here? Maybe just `ClientCredentials` or `ClienCredentialsConfig`?